pkg/app/app.go

```diff
@@ -121,7 +125,13 @@ func (c *App) Handler() http.Handler {
 	rootRouter := mux.NewRouter()
 	rootRouter.PathPrefix("/").Handler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		c.AddCorsHeaderIfNeeded(w, r)
-		router.ServeHTTP(w, r)
+		if r.Method == "OPTIONS" {
+			w.Header().Add("Allow", allowedMethods)
```

Comment:
How about using r.Method instead of allowedMethods since it also allows the methods that is not requested.

Reply:
According to the [specification](https://httpwg.org/specs/rfc9110.html#field.allow), `Allow` header contains all request methods the server supports.
 While `r.Method` here always results in "OPTIONS", I think `allowedMethods` would be appropriate for the spec.

Reply:
Acknowledged, thanks!